README.md

```diff
@@ -3,6 +3,7 @@
 [![Build Status](https://github.com/styrainc/regal/workflows/Build/badge.svg?branch=main)](https://github.com/styrainc/regal/actions)
 ![OPA v0.68.0](https://openpolicyagent.org/badge/v0.68.0)
 [![codecov](https://codecov.io/github/StyraInc/regal/graph/badge.svg?token=EQK01YF3X3)](https://codecov.io/github/StyraInc/regal)
+[![Downloads](https://img.shields.io/github/downloads/styrainc/regal/total.svg)](https://github.com/StyraInc/regal/releases)
```

Comment:
Hm I wonder if this really is interesting to anyone looking at the README. But I suppose it doesn't hurt -- since going through homebrew also just downloads the github release artifact, the number should be fairly accurate.

Reply:
It's mostly just to show that Regal is an established project. 

The downloads should be fairly correct, although they don't account for https://github.com/StyraInc/regal/pkgs/container/regal